platform/ios/src/MGLAPIClient.m

```diff
@@ -76,8 +76,9 @@ - (void)cancelAll {
 #pragma mark Utilities
 
 - (NSURLRequest *)requestForEvents:(NS_ARRAY_OF(MGLMapboxEventAttributes *) *)events {
-    NSString *url = [NSString stringWithFormat:@"%@/%@?access_token=%@", self.baseURL, MGLAPIClientEventsPath, [MGLAccountManager accessToken]];
-    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:[NSURL URLWithString:url]];
+    NSString *path = [NSString stringWithFormat:@"%@?access_token=%@", MGLAPIClientEventsPath, [MGLAccountManager accessToken]];
```

Comment:
In the future, we should consider making the user default hold only the hostname, so that we can built the URL more robustly with NSURLComponents. (For example, we’d set the `scheme` to `https`, no questions asked.)